dissect/target/plugins/apps/webservers/nginx.py

```diff
@@ -0,0 +1,81 @@
+import re
+from zoneinfo import ZoneInfo, ZoneInfoNotFoundError
+from datetime import datetime, timezone
+
+from dissect.target import plugin
+from dissect.target.helpers.fsutil import TargetPath, open_decompress
+from dissect.target.plugins.apps.webservers.webservers import WebserverRecord
+
+
+class NginxPlugin(plugin.Plugin):
+    __namespace__ = "nginx"
+
+    def __init__(self, target):
+        super().__init__(target)
+        self.LOGS_DIR_PATH = self.get_log_paths()
+        self.target_timezone = target.datetime.tzinfo
+
+    @plugin.internal
+    def get_log_paths(self):
+
+        logs_dir = self.target.fs.path("/var/log/nginx")
+        if logs_dir.exists():
+            return [self.target.fs.path(p) for p in logs_dir.glob("access.log*")]
```

Comment:
Would you not rather combine this with possible output of parsing the config file?